frontend/src/components/ThemeSwitcher.vue

```diff
@@ -0,0 +1,21 @@
+<script setup lang="ts">
+import { useDark, useToggle } from '@vueuse/core';
+import { SunIcon, MoonIcon } from '@heroicons/vue/solid';
+
+const isDark = useDark();
+const toggleDark = useToggle(isDark);
+
+const buttonClick = () => {
+  toggleDark();
+};
+</script>
+
+<template>
+  <button
+    class="fixed right-0 bottom-0 mb-4 mr-4 block p-1 rounded-md opacity-75 hover:opacity-100 transition-[opacity,box-shadow] focus:ring-2 focus:outline-none"
```

Comment:
To improve code readability, can we make sure <100 columns for each line?

Reply:
Okay, let me configure my formatter

Reply:
Fixed in https://github.com/kaichiachen/Policy-driven-data-protection/pull/2/commits/1707a2a3edb31d8e9f976ee7a8e1505ba9cddb62

Reply:
Thanks